contracts/storage/LibAppStorage.sol

```diff
@@ -9,9 +9,9 @@ import "../libraries/LibTypes.sol";
 struct AppStorage {
     IERC20 sarcoToken;
     ICurses curses;
-    // The amount to be taken from the embalmer each time a protocol fee should
-    // be collected
-    uint256 protocolFee;
+    // The percentage (i.e. 1 = 1%) of a sarcophagus' total digging fees that will be collected on
+    // createSarcophagus and rewrapSarcophagus, paid by the embalmer
+    uint256 protocolFeeBasePercentage;
```

Comment:
It just occurred to me -- it might be a good idea to also have this explanation in the natspec of `getProtocolFeeBasePercentage`. Just so peeps browsing the contract have a more intuitive location to find a description